### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gegere**
Improving directory list to help with initial setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,17 +18,15 @@ If you use this to further your career, or put money in your pocket, and would l
...
 
-You can also copy the libraries and models directories into your third_party/ion_auth folder.  For example, copy  to /application/third_party/ion_auth/.  The directory structure would be:
+You can also copy the libraries and models directories into your third_party/ion_auth folder.  For example, copy  to /application/third_party/ion_auth/. The directory structure would be:
 
     controllers/Auth.php
-    views/
-    third_party/ion_auth/libraries/Ion_auth.php
-    third_party/ion_auth/libraries/Bcrypt.php
-    third_party/ion_auth/models/Ion_auth_model.php
+    libraries/Ion_auth.php
```
Comment (benedmunds) [inline README.md]: These are incorrect paths for third_party setup

Comment (benedmunds) [reply README.md]: Maybe add two separate directory structure examples?

Comment (gegere) [reply README.md]: I understand this repo is your baby. Simply trying to help to make things more clear. I am using this directory structure (if I followed the Readme there would be an additional **third_party** in the **third_party**) Hopefully, this screenshot will help you understand the change request further.

Ion Auth Directory Structure

**Comment by benedmunds** [comment]
No need to close the PR, I like the idea of making this more clear.

Is there a way to include this screenshot in the README? That actually might be the easiest way to show this.

**Comment by gegere** [comment]
There sure is a way to add images to the README. Many repo have a assets directory or you can host files out of the repo but I think in the repo is nice for edits / updates from the community. I'll update my initial pull request and see what you think.

**Comment by gegere** [comment]
I fugged this, oops. Opening a new pull.

**Pull request closed without merging**